sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -56,17 +56,32 @@ PRINT_CONFIG_MSG("USE_AIRSPEED_MS45XX automatically set to TRUE")
 #define MS45XX_I2C_DEV i2c2
 #endif
 
-/** Sensor I2C slave address (defaults 0x50, 0x6C and 0x8C) */
+/** Sensor I2C slave address (existing defaults 0x50, 0x6C and 0x8C)
+ */
 #ifndef MS45XX_I2C_ADDR
 #define MS45XX_I2C_ADDR 0x50
 #endif
 
-/** MS45xx pressure range in psi.
- * The sensor is available in 1, 2, 5, 15, 30, 50, 100, 150 psi ranges.
- * 1 psi max range should be ~100m/s max airspeed.
+/** MS45xx sensors pressure output type can be in PSI or InH2O, as defined in the datasheet
+ *  if not defined to be MS45XX_PRESSURE_OUTPUT_TYPE_InH2O then PSI is used
+ * */
+#ifndef MS45XX_PRESSURE_OUTPUT_TYPE_InH2O
+#define MS45XX_PRESSURE_OUTPUT_TYPE_InH2O 1
```

Comment:
is this really PSI by default ?

Reply:
Airspeed works and MS4515DO datasheet mentioned it this way.

BTW here how we came to the "magic values" deducted thx to FvT in datasheed.
![parrotdiscoairspeed](https://user-images.githubusercontent.com/483944/43407974-70e43fea-9420-11e8-84a6-15ae3fec19eb.jpg)